App/Seller/Sources/Seller/UI/MessagesView/MyMessagesView.swift

```diff
@@ -0,0 +1,194 @@
+// system
+import SwiftUI
+
+// Internal
+
+// 3rd
+import OptionalAPI
+
+
+protocol SizingFrameType {
+    
+    static var instance: SizingFrameType { get }
+    
+    var size: CGSize { get }
+}
+
+#if os(macOS)
+import AppKit
+public typealias PlatformScreen = NSScreen
+
+extension NSScreen: SizingFrameType {
+    
+    static var instance: SizingFrameType {
+        NSScreen.main!
+    }
+    
+    public var size: CGSize {
+        frame.size
+    }
+}
+
+#else
+import UIKit
+public typealias PlatformScreen = UIScreen
+
+extension UIScreen: SizingFrameType {
+    static var instance: SizingFrameType {
+        UIScreen.main
+    }
+    
+    public var size: CGSize {
+        bounds.size
+    }
+}
+
+#endif
+
+struct MessagesView: View {
+    @ObservedObject private var viewModel = MyMessagesViewModel()
+    
+    var body: some View {
+        VStack(spacing: 0) {
+            NavigationView(viewModel: viewModel)
+                .frame(height: 77)
+            ScrollView {
+                VStack(spacing: 17) {
```

Comment:
why 77 and 17? What do you think about making those relative to something, like: let navigationViewHeigh = screenHeigh/2? Is it possible?
I see below that some values are reused. What do you think about extracting those to variables and reuse?

Reply:
It is possible with geometry reader, although at this point it is hard to specify the best approach imo, as those relative values should be adjusted to size classes and dimensions usually.

Magic numbers usually appear when they are not repeating in a context in this scenario, I am planning to add task for creating Theme for fonts, colors, and some reusable paddings.

Reply:
👍